Gemfile

```diff
@@ -11,7 +11,14 @@ else
   gem "rails", "~> #{rails_version}"
 end
 
-gem "rack-test"
+group :test do
+  if RUBY_VERSION == "1.9.3"
```

Comment:
... and once this is merged into master, you can publish to rubygems with `rake release`.